src/cljs/commiteth/handlers.cljs

```diff
@@ -64,19 +63,15 @@
 
 
 (reg-event-db
- :set-active-page
- (fn [db [_ page]]
-   (assoc db :page page)))
-
-(reg-event-db
-  :set-bounty-page-number
+  :set-active-page
   (fn [db [_ page]]
-    (assoc db :bounty-page-number page)))
+    (assoc db :page page
```

Comment:
What is `:page` used for?

Reply:
It's an active page keyword, e.g. `:bounties` or `:activity`.

Reply:
Yeah, sorry, I meant what is the difference between `:page` and `:page-number`, that both are being used here.

Reply:
You're right - naming is slightly bad, `:page` should be `:tab` I guess. So it's confusing.